README.md

```diff
@@ -142,14 +77,91 @@ npm run lint
+| `WEBTRENDS_TRACKING_ID`            | [Webtrends](https://www.webtrends.com/) tracking id                                 |                          |
+| `HOTJAR_TRACKING_ID`               | [Hotjar](https://www.hotjar.com/) tracking id                                       |                          |
+| `FONT_CDN`                         | Base url where the font is served                                                   | /                        |
+| `STATIC_CDN`                       | Base url where all other assets are served                                          |                          |
+| `LOG_LEVEL`                        | Level of logging to user                                                            | warn                     |
+| `WDIO_BASEURL`                     | base URL for webdriver to use for acceptance tests                                  | http://localhost:${PORT} |
+| `WDIO_SCREENSHOTPATH`              | path where webdriver screenshots are saved                                          |                          |
+| `WDIO_REPORTPATH`                  | path where webdriverio test runner reports are saved                                |                          |
+| `BROWSERSTACK_USERNAME`            | Browserstack username                                                               |                          |
+| `BROWSERSTACK_ACCESS_KEY`          | Browserstack access key                                                             |                          |
+| `TRAVIS_BUILD_NUMBER`              | The number of the current Travis build                                              |                          |
+| `TRAVIS_JOB_NUMBER`                | The number of the current Travis job                                                |                          |
+| `DISABLE_FEEDBACK`                 | Whether to disable feedback. On by default. Set this to `true` to disable           |                          |
+| `FEEDBACK_API_BASEURL`             | Base feedback endpoint                                                              |                          |
+| `FEEDBACK_API_KEY`                 | Key for feedback API if needed                                                      |                          |
+| `FEEDBACK_TIMEOUT`                 | Timeout before the request to the API fails                                         | 5000                     |
+| `APPINSIGHTS_INSTRUMENTATIONKEY`   | Application insights instrumentation key                                            |                          |
+| `CONNECTINGTOSERVICES_BASEURL`     | Base URL for connecting to services application                                     | /                        |
+| `CONTENTSTORE_TYPE`                | Operational mode for content store (`rest` or `file`)                               | file                     |
```

Comment:
We now actually support a third type, `fallback`, which looks for that record via REST then falls back to file. Should add that in as it was previously missed.